### PR TITLE
Adds sortable functionality to detail view fields

### PR DESCRIPTION
**Pull request by tonytlwu**
Ref. https://github.com/Fliplet/fliplet-studio/issues/6126

## Screenshot

image

### CODE REVIEW

File: css/interface.css [hunk 1]
```diff
@@ -425,7 +425,8 @@ header .btn,
 }
 
 .panel-title .fa {
-  cursor: pointer;
+  cursor: -webkit-grab;
+  cursor: -moz-grab;
```
Comment (squallstar) [inline css/interface.css]: no `grab` ?

Comment (hcarneiro) [reply css/interface.css]: Same on line `920`.

Comment (tonytlwu) [reply css/interface.css]: Apparently not. I'm following the existing patterns, which seem to be working quite well.

Comment (hcarneiro) [reply css/interface.css]: Weird, but ok

**Pull request merged**